### ISSUE TITLE
consistent line spacing in GUI?

### ISSUE THREAD

**Issue by rsignell-usgs** [intake/xrviz#8]
Is it possible to use the consistent line spacing for both regions 1 and 2 shown here?
![2019-06-17_9-02-55](https://user-images.githubusercontent.com/1872600/59606529-60f0a080-90df-11e9-9e98-61d37d29cf4a.png)

It might seem silly, but I think these style issues are important for the user experience.

**Comment by martindurant** [comment]
I agree, although it may be premature if we think the visual components might yet move around. I think we will probably need some pyviz people to help out with final styling and alignment.
